content/version/1/4/code-of-conduct.gu.md

```diff
@@ -0,0 +1,76 @@
++++
+version = "1.4"
+aliases = ["/version/1/4/gu"]
++++
+
+# ફાળકોના ઠરાવની આચારસંહિતા
```

Comment:
Instead of using `ફાળકો`, `ફાળો આપનારાઓ ` is more readable.
But as this is a legal document `ફાળકો` is still fine for me.

Reply:
yes, ફાળો આપનારાઓ is more readable than ફાળક as a single word, but it is not grammatically incorrect. Also, this change will increase the length of the document and make it more unreadable further down the line, because the noun ફાળક is used and inflected many times later in the document. 

Also, the discomfort for the word, may also be due to its unfamiliarity. But words become comfortable as they are used more which leads to more familiarity (and hence readability).